orm/query.go

```diff
@@ -482,60 +506,60 @@ func (q *Query) WhereOr(condition string, params ...interface{}) *Query {
 
 // WhereGroup encloses conditions added in the function in parentheses.
 //
-//    q.Where("TRUE").
-//    	WhereGroup(func(q *pg.Query) (*pg.Query, error) {
-//    		q = q.WhereOr("FALSE").WhereOr("TRUE").
-//    		return q, nil
-//    	})
+//	q.Where("TRUE").
+//		WhereGroup(func(q *pg.Query) (*pg.Query, error) {
+//			q = q.WhereOr("FALSE").WhereOr("TRUE").
+//			return q, nil
+//		})
```

Comment:
Could you undo all these whitespace changes please?

Reply:
sure, looks like it came from vscode